---
source: src/validation/tests/naming_validation_test.rs
expression: res
---
SyntaxError { message: "__U1 can not be used as a name because it is a built-in datatype", range: [SourceRange { range: 5..9 }], err_no: type__invalid_name }
SyntaxError { message: "__U1: Ambiguous datatype.", range: [SourceRange { range: 5..9 }, SourceRange { range: 0..0 }], err_no: duplicate_symbol }

